dev/bots/test.dart

```diff
@@ -528,6 +529,33 @@ Future<Null> _runFlutterAnalyze(String workingDirectory, {
   );
 }
 
+Future<Null> _verifyNoTestPackageImports(String workingDirectory) async {
```

Comment:
Why not put this in `flutter_tools/test/forbidden_imports_test.dart`?

Reply:
That only checks `flutter_tools`. This checks the whole repo.